### PR TITLE
Stable fixed for merge

### PR DESCRIPTION
**Pull request by joernu76**
**Purpose of PR?**:

Fixes #

**Does this PR introduce a breaking change?**

**If the changes in this PR are manually verified, list down the scenarios covered:**:

**Additional information for reviewer?** :
_Mention if this PR is part of any design or a continuation of previous PRs_

**Does this PR results in some Documentation changes?**
_If yes, include the list of Documentation changes_

**Checklist:**
- [ ] Bug fix. Fixes #<issue number>
- [ ] New feature (Non-API breaking changes that adds functionality)
- [ ] PR Title follows the convention of  `<type>: <subject>`
- [ ] Commit has unit tests

### CODE REVIEW

**Comment by joernu76** [comment]
I had to create a merge commit to resolve conflicts. Is this the way?

**Comment by ReimarBauer** [comment]
Can you send the existing merge_stable_to_develop branch again for PR. 
This should be still around.

**Comment by ReimarBauer** [comment]
trying
https://github.com/Open-MSS/MSS/pull/2493

File: tests/_test_mscolab/test_chat_manager.py [hunk 1]
```diff
@@ -65,9 +71,22 @@ def test_edit_messages(self):
...
             message = self.cm.add_message(self.user, 'some test example message',
-                                          self.operation_name, message_type=MessageType.TEXT,
+                                          self.operation.id, message_type=MessageType.TEXT,
                                           reply_id=None)
             assert 'some test example message' in message.text
             self.cm.delete_message(message.id)
             message = Message.query.filter(Message.id == message.id).first()
             assert message is None
+
+    def test_add_attachment(self):
```
Comment (ReimarBauer) [inline tests/_test_mscolab/test_chat_manager.py:81]: this was moved by Aryan in current develop.

git log -S test_add_attachment

https://github.com/Open-MSS/MSS/pull/2431/files#diff-1444e9a8f44488bf14befb677e2c489087ab68fdc164ecb3426ad2c5b4649d10

**Pull request closed without merging**